src/Data/Functor/Classes/Compat.hs

```diff
@@ -1,5 +1,3 @@
-#include "free-common.h"
-#ifdef LIFTED_FUNCTOR_CLASSES
 {-# LANGUAGE Safe #-}
 module Data.Functor.Classes.Compat (
```

Comment:
I'm not sure how necessary this module is nowadays, given that it no longer defines any compatibility shims. Perhaps I will just fold this into `Control.Applicative.Free`.